### PR TITLE
`trigger.name` & `trigger.useCachedMetrics` docs

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubalik@gmail.com>

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

### CODE REVIEW

File: content/docs/2.9/concepts/scaling-deployments.md [hunk 1]
```diff
@@ -210,6 +210,8 @@ Starting from Kubernetes v1.18 the autoscaling API allows scaling behavior to be
 Trigger fields:
 - **type**: The type of trigger to use. (Mandatory)
 - **metadata**: The configuration parameters that the trigger requires. (Mandatory)
+- **name**: An optional name for this trigger. This value can be used to easily distinguish this specific trigger and it's metrics when consuming [Prometheus metrics](../operate/prometheus.md). (Optional)
+- **useCachedMetrics**: This feature enables caching of metric values during polling interval (as specified in `.spec.pollingInterval`). Kubernetes (HPA controller) asks for a metric every few seconds (as defined by `--horizontal-pod-autoscaler-sync-period`, usually 15s), then is this request routed to KEDA Metrics Server, that by default queries the scaler and reads the metric values. Enabling this feature changes this behavior, KEDA Metris Server tries to read metric from the cache first. This cache is being updated periodically during the polling interval. Enabling this feature can significantly reduce the load on the scaler service. This is an [experimental feature](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md#experimental-features). (Values: `false`, `true`, Default: `false`, Optional)
```
Comment (tomkerkhove) [inline content/docs/2.9/concepts/scaling-deployments.md:214]: Let's move the details to a dedicated section as it's too much here and we can add a experimental banner there:
```suggestion
- **useCachedMetrics**: ([Experimental feature](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md#experimental-features)) Enables caching of metric values during polling interval (as specified in `.spec.pollingInterval`). For more information, see ["Caching Metrics (Experimental)"](#<something>) (Values: `false`, `true`, Default: `false`, Optional)
```

Bonus points if that new section has a quick visual overview of how it works.

File: content/docs/2.9/concepts/scaling-deployments.md [hunk 2]
```diff
@@ -210,6 +210,8 @@ Starting from Kubernetes v1.18 the autoscaling API allows scaling behavior to be
 Trigger fields:
 - **type**: The type of trigger to use. (Mandatory)
 - **metadata**: The configuration parameters that the trigger requires. (Mandatory)
+- **name**: An optional name for this trigger. This value can be used to easily distinguish this specific trigger and it's metrics when consuming [Prometheus metrics](../operate/prometheus.md). (Optional)
```
Comment (tomkerkhove) [inline content/docs/2.9/concepts/scaling-deployments.md:213]: Optional is already refered to at the end.
```suggestion
- **name**: Name for this trigger. This value can be used to easily distinguish this specific trigger and its metrics when consuming [Prometheus metrics](../operate/prometheus.md). (Optional)
```

Also, what happens if it's not specified; what value will it use?

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

File: content/docs/2.9/concepts/scaling-deployments.md (hunk 1 above)
Comment (zroubalik) [reply content/docs/2.9/concepts/scaling-deployments.md:214]: Where should I put the extra info? I don't the current location is that bad.

Comment (tomkerkhove) [reply content/docs/2.9/concepts/scaling-deployments.md:214]: For majority of end-users this is TMI; I'd add a new h3 for this with details similar to https://keda.sh/docs/2.8/concepts/scaling-deployments/#pause-autoscaling

**Review by tomkerkhove — APPROVED** [review]
Perfect, thanks!

**Pull request merged**